recipes/spam/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "spam" %}
+{% set version = "0.6.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gricad-gitlab.univ-grenoble-alpes.fr/ttk/spam/-/archive/62a7c325dbea0d90314ba19a0ac76a61a23691f4/spam-62a7c325dbea0d90314ba19a0ac76a61a23691f4.tar
```

Comment:
The automation bot will not be able to help you at all with commits... and custom gitlabs are also somewhat under-researched.

Given this is registered pypi: can you advocate for getting sdists uploaded there?

Failing that, can the upstream `version-0.6.1.1` tag be used?